beets/ui/__init__.py

```diff
@@ -924,7 +924,8 @@ def _raw_main(args, lib=None):
     # Special case for the `config --edit` command: bypass _setup so
     # that an invalid configuration does not prevent the editor from
     # starting.
-    if subargs[0] == 'config' and ('-e' in subargs or '--edit' in subargs):
+    if subargs and subargs[0] == 'config' \
+       and ('-e' in subargs or '--edit' in subargs):
```

Comment:
Good catch. I guess it's visually aligned with the start of the expression on the previous line. In any case, feel free to fiddle with it using your newfound commit powers. :smiley: